node/external/blockAPI/baseBlock.go

```diff
@@ -106,13 +106,19 @@ func (bap *baseAPIBlockProcessor) prepareAPIMiniblock(miniblock *block.MiniBlock
 		DestinationShard: miniblock.ReceiverShardID,
 	}
 	if withTxs {
-		bap.getAndAttachTxsToMbByEpoch(mbHash, miniblock, epoch, miniblockAPI)
+		firstProcessed := int32(0)
```

Comment:
since this is correct for the intrashard miniblock, let's rename prepareAPIMiniblock to something like prepareAPIMiniblockForIntrashard and extractMbsFromBatch to something like extractIntrashardMbsFromBatch